### PR TITLE
JSON to YAML for external LoadBalancer

### PR DESCRIPTION
**Pull request by programmer04**
Make this page consistent:
- YAML definitions for an object instead of JSON,
- apply style guide rules,
- remove obsolete fragments.

### CODE REVIEW

File: content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 1]
```diff
@@ -178,8 +166,8 @@ after v1.7. Please update existing applications to use the fields directly.
 `service.beta.kubernetes.io/external-traffic` annotation has a different set of values
 compared to the `service.spec.externalTrafficPolicy` field. The values match as follows:
 
-* "OnlyLocal" for annotation <-> "Local" for field
-* "Global" for annotation <-> "Cluster" for field
+* OnlyLocal for annotation <-> Local for field
```
Comment (sftim) [inline content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md]:
```suggestion
* OnlyLocal for annotation ⭤ Local for field
```

File: content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 2]
```diff
@@ -178,8 +166,8 @@ after v1.7. Please update existing applications to use the fields directly.
...
+* Global for annotation <-> Cluster for field
```
Comment (sftim) [inline content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md]:
```suggestion
* Global for annotation ⭤ Cluster for field
```

File: content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md]: However: https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#feature-availability is outdated. It'd be fine to drop this entire fragment.

File: content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md [hunk 3]
```diff
@@ -124,40 +118,34 @@ service spec (supported in GCE/Google Kubernetes Engine environments):
...
-effect when type is set to "LoadBalancer" and externalTrafficPolicy is set
-to "Local".
+(numeric port number) for the service. If not specified, `healthCheckNodePort` is
+created by the service API backend with the allocated `nodePort`. It will use the
+user-specified `nodePort` value if specified by the client. It only has an
+effect when type is set to LoadBalancer and `externalTrafficPolicy` is set
+to Local.
 
-This feature can be activated by setting `externalTrafficPolicy` to "Local" in the
+This feature can be activated by setting `externalTrafficPolicy` to Local in the
```
[the remark this answers is not in the document]
Comment (programmer04) [reply content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md]: Fixed everywhere where I spotted this

File: content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md (hunk 2 above)
Comment (programmer04) [reply content/en/docs/tasks/access-application-cluster/create-external-load-balancer.md]: Removed

**Comment by programmer04** [comment]
Howdy @sftim, thanks for the review, I applied all suggestion. Please check

**Comment by sftim** [comment]
These changes are clearly improving the existing content.
/lgtm

**Pull request merged**